### DISCUSSION TITLE
HmIP-WLAN-HAP in Raspberrymatic einbinden.

### DISCUSSION THREAD

**Discussion by paulpf** [OpenCCU/OpenCCU#2595] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich bin mir nicht sicher, was der aktuelle Stand ist und was die Zukunft verspricht. Die Recherchen im Netz geben mir eine fast eindeutige Antwort. Nichtsdestotrotz würde ich gerne diese Frage in die Runde stellen (bitte verzeiht mir, falls es für manche zu viel ist): wird es rein technisch irgendwann möglich sein die HmIP-WLAN-HAP als Repeater unter RaspberryMatic einzubinden?

Vielen Dank für die Antwort!

**Comment by jens-maus** [comment, marked as answer]
>... wird es rein technisch irgendwann möglich sein die HmIP-WLAN-HAP als Repeater unter RaspberryMatic einzubinden?

Das **wird** erst dann möglich sein wenn eQ3 als Hersteller das auch entsprechend unterstützt und für diesen `HmIP-WLAN-HAP` eine entsprechende alternative Firmware zur Verfügung stellt wie für die LAN-gebundenen HAP. Das ist aber aktuell nicht der Fall und soweit ich weiss ist das auch nicht geplant. Und übrigens ist das auch keine Limitation von RasbperryMatic, sondern man kann einen `HmIP-WLAN-HAP` auch nicht – aus den selben Gründen – an eine CCU3 mit der CCU-Firmware von eQ3 anlernen.

**Reply by paulpf** [reply]
Das habe ich mir fast gedacht, dass es nicht an Raspberrymatic liegt. Mir fehlte lediglich die technische Kenntnis des HmIP-WLAN-HAP-Moduls bzw. die Business-Strategie von eQ3. Danke für die schnelle Antwort, Jens!

**Reply by jp112sdl** [reply]
@paulpf Frag doch mal bei eQ-3 nach. Im besten Fall bekommst du eine "Wir arbeiten bereits an... um zukünftig..." Antwort.

**Reply by paulpf** [reply]
Ich glaube das mache ich wirklich... Ich habe mir letzte Woche so ein Teil gekauft, in der Hoffnung, dass es auf Anhieb funktioniert... Zwei der kabelgebunden HAPs funktionierten tadellos... Nur die Position war bei einem nicht optimal. WLAN- Modul hätte das Problem gelöst. Mein Fehler.
